cypress/fixtures/dummyEventResponses.json

```diff
@@ -0,0 +1,13 @@
+{
+    "data": [
+        {
+            "action_id": 22,
+            "response_date": "1111 11 11, 11:11",
```

Comment:
I dislike the use of this very weird date. Why the year 1111, and not something more alike to what would occur in live data? It's also not using the correct ISO format. The API delivers dates like `2021-01-21T18:30:00+00:00`, and the dummy data should mimic that.